ffmpeg/lpms_ffmpeg.c

```diff
@@ -841,6 +841,15 @@ static int open_input(input_params *params, struct input_ctx *ctx)
   ctx->ic = ic;
   ret = avformat_find_stream_info(ic, NULL);
   if (ret < 0) dd_err("Unable to find input info\n");
+  if (AV_HWDEVICE_TYPE_CUDA == params->hw_type) {
+    for( int i = 0; i < ic->nb_streams; i++ ) {
+      if( ic->streams[i]->codec->codec_type == AVMEDIA_TYPE_VIDEO &&
+          ic->streams[i]->codec->pix_fmt != AV_PIX_FMT_YUV420P ) {
```

Comment:
Think this can go into `open_video_decoder` ?  Maybe early on in this block?
 https://github.com/livepeer/lpms/blob/285e87fe283a4844192275d3c9717ef4dbaba24b/ffmpeg/lpms_ffmpeg.c#L752-L757

Reply:
Moved